src/core_plugins/kibana/public/visualize/editor/editor.html

```diff
@@ -66,12 +66,7 @@
     </div>
   </div>
 
-  <visualize
-    ng-if="!chrome.getVisible()"
-    saved-obj="savedVis"
-    ui-state="uiState"
-    time-range="timeRange"
-  />
+  <div class="visualize" />
```

Comment:
The `visualize` class has it's own style and now it's visible also inside the editor, moving everything on the bottom of the page.
It's better to add an `ng-if` to remove the div when not used
<img width="1439" alt="screen shot 2018-07-31 at 12 17 25" src="https://user-images.githubusercontent.com/1421091/43453975-e02920dc-94bb-11e8-98a9-c2063246342a.png">